src/test/java/com/issoft/training/tests/AutomationPracticeShoppingTests.java

```diff
@@ -0,0 +1,188 @@
+
+import static com.issoft.training.config.ConfProperties.getProperty;
+
+@Listeners({ AllureListener.class })
+@Epic("Regression tests")
+@Feature(value = "Automation practice - Shopping tests")
+public class AutomationPracticeShoppingTests extends TestBase {
+    // page objects
+    CreateAccountPage createAccountPage;
+    AccountInformationPage accountInformationPage;
+    LoginAccountPage loginAccountPage;
+    MyAccountPage myAccountPage;
+    MyWishlistsPage myWishlistsPage;
+
+    @Test(priority = 0, groups = {"accountTests", "checkinTests"},
+            description = "Verify the ability to create an account.")
+    @Severity(SeverityLevel.NORMAL)
+    @Description("Test Description: Verify the ability to create an account.")
+    @Story("Story name: Account creation.")
+    public void createMyAccountTest() {
+        // test AP-1
+        driver.get(getProperty("url_automationpractice_loginpage"));
+        Assert.assertEquals(driver.getTitle(), Constants.TITLE_OF_ACCOUNT_PAGE);
+        createAccountPage = new CreateAccountPage(driver);
+        accountInformationPage = new AccountInformationPage(driver);
+
+        createAccountPage.inputEmail(getNewEmailAddress());
+        getScreenShotFile(driver, Constants.INPUT_EMAIL_SCREEN);
+        createAccountPage.clickCreateAccountBtn();
+        Assert.assertTrue(driver.findElement(Constants.HEADER_PERSONAL_INFO_PAGE).isDisplayed());
+
+        accountInformationPage.clickCustomerGenderBtn();
+        accountInformationPage.inputCustomerFirstName(ConfProperties.getProperty("firstname"));
+        accountInformationPage.inputCustomerLastName(ConfProperties.getProperty("lastname"));
+        accountInformationPage.inputCustomerPasswd(ConfProperties.getProperty("passwd"));
+        accountInformationPage.inputUniformDays(ConfProperties.getProperty("days"));
+        accountInformationPage.inputUniformMonth(ConfProperties.getProperty("month"));
+        accountInformationPage.inputUniformYear(ConfProperties.getProperty("year"));
+        accountInformationPage.inputAddressFirstName(ConfProperties.getProperty("firstname"));
+        accountInformationPage.inputAddressLastName(ConfProperties.getProperty("lastname"));
+        accountInformationPage.inputAddressCompanyName(ConfProperties.getProperty("companyname"));
+        accountInformationPage.inputCompanyAddress1(ConfProperties.getProperty("address"));
+        accountInformationPage.inputCompanyCity(ConfProperties.getProperty("city"));
+        accountInformationPage.inputAddressState(ConfProperties.getProperty("state"));
+        accountInformationPage.inputAddressPostCode(ConfProperties.getProperty("postcode"));
+        accountInformationPage.inputCompanyCountry(ConfProperties.getProperty("country"));
+        accountInformationPage.inputPhoneMobile(ConfProperties.getProperty("phone"));
+        accountInformationPage.inputAddressAlias(ConfProperties.getProperty("alias"));
+        accountInformationPage.clickSubmitAccountBtn();
```

Comment:
Please move it to the AccountInformationPage class as a new method. It is could that not one your test need to fill all those fields. It will be great if those actions can be reused. 

Also those steps overload test which is not godd

Reply:
Created a new method.